src/graph/interface/tensor.cpp

```diff
@@ -109,6 +109,20 @@ static void tensor_free(void *p, const engine_t *eng) {
     }
 }
 
+static void *host_tensor_malloc(
```

Comment:
why do we need to specific function for it?

Reply:
The tensor_malloc function will create cpu sycl memory when sycl runtime is enabled. This host_tensor_malloc function will always create host memory even when cpu sycl runtime is enabled.